shell/browser/api/electron_api_url_loader.cc

```diff
@@ -427,8 +427,7 @@ void SimpleURLLoaderWrapper::OnDataReceived(base::StringPiece string_piece,
   auto array_buffer = v8::ArrayBuffer::New(isolate, string_piece.size());
   auto backing_store = array_buffer->GetBackingStore();
   memcpy(backing_store->Data(), string_piece.data(), string_piece.size());
-  Emit("data", array_buffer);
-  std::move(resume).Run();
+  Emit("data", array_buffer, base::AdaptCallbackForRepeating(std::move(resume)));
```

Comment:
Lint is failing:

```patch
$ node ./script/lint.js && npm run lint:clang-format && npm run lint:docs
--- a/shell/browser/api/electron_api_url_loader.cc
+++ b/shell/browser/api/electron_api_url_loader.cc
@@ -427,7 +427,8 @@
   auto array_buffer = v8::ArrayBuffer::New(isolate, string_piece.size());
   auto backing_store = array_buffer->GetBackingStore();
   memcpy(backing_store->Data(), string_piece.data(), string_piece.size());
-  Emit("data", array_buffer, base::AdaptCallbackForRepeating(std::move(resume)));
+  Emit("data", array_buffer,
+       base::AdaptCallbackForRepeating(std::move(resume)));
 }
 
 void SimpleURLLoaderWrapper::OnComplete(bool success) {
```

Reply:
@zcbenz Thanks so much! Ran a lint check and made sure this was fixed 🙇‍♀️